presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -1734,6 +1735,8 @@ public void testGroupingSetsAliasedGroupingColumns()
     @Test
     public void testGroupingSetMixedExpressionAndColumn()
     {
+        featuresToTest(DATE_TYPE);
```

Comment:
It's hard to assume that `testGroupingSetMixedExpressionAndColumn` tests ... "date data type". It does not. It tests something like "grouping set with ...." and "date data type" is only technical requirement for a test.
Let's separate these two aspects for clarity. For example you could have two annotations:
```
@TestedFeatures(GROUPING_SET)
@RequiredFeatures(DATE_TYPE)
```
ideally, we should refactor the tests so that `@RequiredFeatures` is never needed. Of course, nothing to do now, but keeping those separate let's us identify where we depart from this ideal state, and improve.